Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -1445,15 +1641,10 @@ static HRESULT __CGContextCreateShadowEffect(CGContextRef context,
 }
 
 template <typename Lambda> // Lambda takes the form void(*)(CGContextRef, ID2D1DeviceContext*)
-static HRESULT __CGContextRenderToCommandList(CGContextRef context,
-                                              _CGCoordinateMode coordinateMode,
-                                              CGAffineTransform* additionalTransform,
-                                              ID2D1CommandList** outCommandList,
-                                              Lambda&& drawLambda) {
-    ComPtr<ID2D1RenderTarget> renderTarget = context->RenderTarget();
-    ComPtr<ID2D1DeviceContext> deviceContext;
-    RETURN_IF_FAILED(renderTarget.As(&deviceContext));
-
+HRESULT __CGContext::DrawToCommandList(_CGCoordinateMode coordinateMode,
```

Comment:
small body vs large body.

---
In reply to: [90589638](https://github.com/Microsoft/WinObjC/pull/1496#discussion_r90589638) [](ancestors = 90589638)